accessibility/css/reveal.scss

```diff
@@ -0,0 +1,1319 @@
+/*!
```

Comment:
I don't think we'll need these files if we already have a compiled `reveal.css` file, unless you've made modifications?